### ISSUE TITLE
Panel field checkboxes long label issue

### ISSUE THREAD

**Issue by monobasic** [getkirby/panel#905]
When texts for panel checkboxes labels getting longer, column layout is not calculated correct any more and the fields overlap the current viewport width.

1-col:
https://www.dropbox.com/s/1nnqu9iiu58krq6/Screenshot%202016-08-23%2010.26.31.png?dl=0

2-col mobile:
https://www.dropbox.com/s/qe3gb0n2qflmq2v/Screenshot%202016-08-23%2010.28.57.png?dl=0

**Comment by monobasic** [comment]
Link to issue @ kirby forum: https://forum.getkirby.com/t/panel-field-checkboxes-long-label-issue/5043

**Comment by lukasbestle** [comment]
Could you please post your blueprint? I have tested this in several browser and long text wraps to the next line inside the checkbox field.

**Comment by texnixe** [comment]
I tested this as well and the text did not wrap ... (Safari and Chrome)

**Comment by lukasbestle** [comment]
Can I have the blueprint you were using?

**Comment by texnixe** [comment]
Sure

```
title: Home
pages: false
fields:
  title:
    label: Title
    type:  text
  text:
    label: Text
    type:  textarea
  checky:
    label: Checkboxes
    type: checkboxes
    options:
      a:a jfdjf dakjfdaks jakjd fasj kfjdsfk jasdkj jakj fakjd kajdf kajf kaj kfajk fjadsö kfjkasj f
      b: aj dkajsf kajsdfkj asdkfj asjdfkasjf  jdkjfkajd ajdfkjadfk jadskfj akjdf kasjf kadjfk kjdf
```

**Comment by monobasic** [comment]
Sorry I don't have blueprint used for testing any more.. basically I just used something like texnixe - static options with very long blindtext.

**Comment by monobasic** [comment]
Now I can reproduce it on my local installation, you have to use even longer values:

`
  checky:
    label: Checkboxes
    type: checkboxes
    options:
      a: a jfdjf dakjfdaks jakjd fasj kfjdsfk jasdkj jakj fakjd kajdf kajf kaj kfajk fjadsö kfjkasj fasdfalkjdsföajsd faöklsjdf aölsdkj faösdkj föalksdj fasd
      b: aj dkajsf kajsdfkj asdkfj asjdfkasjf  jdkjfkajd ajdfkjadfk jadskfj akjdf kasjf kadjfk kjdf asjfdkasdöflajksdöflkjas dflajk sdöflkaj sdöfklajs döfakj asdökjf a

`

**Comment by lukasbestle** [comment]
Ah, I see. I was testing with the `checkbox` field (singular) where the text wraps instead of being cut off. I can reproduce this now.

**Comment by bastianallgeier** [comment]
I just fixed this on the develop branch and it will be included in 2.4

**Comment by monobasic** [comment]
Many thanks :)
